apps/api-extractor/src/generators/ExcerptBuilder.ts

```diff
@@ -195,10 +195,10 @@ export class ExcerptBuilder {
       capturedTokenRange.startIndex = excerptStartIndex;
 
       // We will assign capturedTokenRange.startIndex to be the index after the last token
-      // that was appended so far. However, if the last appended token was a separator,
-      // then omit it from the range.
+      // that was appended so far. However, if the last appended token was a separator and
+      // there is no additional spaces, omit it from the range.
       let excerptEndIndex: number = excerptTokens.length;
-      if (state.lastAppendedTokenIsSeparator) {
+      if (state.lastAppendedTokenIsSeparator && excerptEndIndex - excerptStartIndex !== 1) {
         excerptEndIndex--;
       }
 
```

Comment:
> Is leading to a returnTypeTokenRange of [4,5), where internally we are seeing [5,5). 
> Is there a parameter that changes the handling of extraneous spaces?

I'm not sure what you're asking about.  What kind of parameter do you mean?

Reply:
> Replicating the issue in a rushStack api-extractor-scenario with a class such as this

If you could add this test case to your PR branch, that would be super helpful. :-)

Reply:
Will do once I understand the reasoning for the difference in output; as atm without any dts rollup options (*trimmedFilePath from https://api-extractor.com/pages/setup/configure_rollup/) declared we are seeing extra spaces in the rollup.d.ts which is leading to the edge case we are seeing. For the code segment above we would end up with the d.ts looking like.
```
someMethod(x : number , y : string ) : boolean ;
```
where as API extractor is expecting the method signature to look more like
```
someMethod(x: number, y: string): boolean;
```

Reply:
Alright more details as this developed and the like.

I added a new api-extractor-test that runs on a predetermined d.ts with additional spaces, which is valid typescript, just different than the output of TSC.

I hope this illuminates the edge case appropriately and is ready for further review.